server/src/test/java/org/alfresco/bm/driver/test/prop/TestPropertyFactoryTest.java

```diff
@@ -157,6 +157,7 @@ public void testDefinitions()
         assertEquals(256, strProp.getMax());
         assertEquals(".*", strProp.getRegex());
         assertEquals(true, strProp.isMask());
+        assertEquals(0, strProp.getIndex());
```

Comment:
I'm not sure how valuable these tests are, but usually it's good to add a message. That way it's easy to debug if it fails.

Reply:
It's probably not worth updating though, since none of the assertions here have messages.